src/openrct2/audio/audio.h

```diff
@@ -92,27 +92,27 @@ enum RCT2_SOUND
     SOUND_SCREAM_4,
     SOUND_SCREAM_5,
     SOUND_SCREAM_6,
-    SOUND_LIFT_3,
+    SOUND_LIFT_VEKOMA,
     SOUND_PURCHASE,
     SOUND_CRASH,
     SOUND_LAYING_OUT_WATER,
     SOUND_WATER_1,
     SOUND_WATER_2,
     SOUND_TRAIN_WHISTLE,
-    SOUND_TRAIN_CHUGGING,
+    SOUND_TRAIN_DEPARTING,
     SOUND_WATER_SPLASH,
-    SOUND_HAMMERING,
+    SOUND_GO_KART_ENGINE,
     SOUND_RIDE_LAUNCH_1,
     SOUND_RIDE_LAUNCH_2,
     SOUND_COUGH_1,
     SOUND_COUGH_2,
     SOUND_COUGH_3,
     SOUND_COUGH_4,
-    SOUND_RAIN_1,
+    SOUND_RAIN,
     SOUND_THUNDER_1,
     SOUND_THUNDER_2,
-    SOUND_RAIN_2,
-    SOUND_RAIN_3,
+    SOUND_TRACK_FRICTION_TRAIN,
+    SOUND_TRACK_FRICTION_WATER,
```

Comment:
Could you clarify this one? You might be right, but I don't immediately recognise the sound.

Reply:
This is used for most of the water rides. Examples include the River Rapids and the Dinghy Slide:
https://github.com/OpenRCT2/objects/blob/72dfb9adc551a0b5a2e173dcf6536402e9a5acb0/objects/rct2/ride/rct2.ding1.json#L32

Exceptions include the river rafts.